src/components/PeoplePage.tsx

```diff
@@ -0,0 +1,123 @@
+import { useEffect, useState } from "react";
+import { Person } from "../types";
+import { getPeople } from "../api";
+import { Loader } from "./Loader";
+import { Link, useParams } from "react-router-dom";
+import classNames from "classnames";
+
+
+export const PeoplePage = () => {
+
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState<boolean>(true);
+  const [error, setError] = useState<string | null>(null);
+
+  const { personSlug } = useParams();
+
+  const personInfo = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+
+  const personName = (name: string | null) => {
+    return people.find(person => person.name === name);
+  }
+
+
+  useEffect(() => {
+   setLoading(true)
+   getPeople()
+   .then(data => {
+    setPeople(data)
+    setLoading(false)
+   })
+   .catch(err => {
+     setError('Failed to load people data')
+     setLoading(false)
+   })
```

Comment:
It's a good practice to handle errors properly in the catch block. Currently, you are just setting the error message to 'Failed to load people data', but you are not using the error object 'err'. It would be better to use the error object to get more detailed information about the error. You could do something like this: `setError(err.message || 'Failed to load people data')`.